201-vm-win-iis-app-ssl/azuredeploy.json

```diff
@@ -0,0 +1,409 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "appStorageType": {
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_ZRS",
+        "Standard_GRS",
+        "Standard_RAGRS",
+        "Premium_LRS"
+      ],
+       "metadata": {
+        "description": "Set this value for the storage account type"
+      }
+    },
+    "appVMName": {
+      "type": "string",
+      "minLength": 1,     
+       "metadata": {
+        "description": "Set this value for the VM name"
+      }
+    },
+    "appVMAdminUserName": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Set this value for the VM admin user name"
+      }
+    },
+    "appVMAdminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Set this value for the VM admin user password"
+      }
+    },
+    "appVMWindowsOSVersion": {
+      "type": "string",
+      "defaultValue": "2012-R2-Datacenter",
+      "allowedValues": [
+        "2008-R2-SP1",
+        "2012-Datacenter",
+        "2012-R2-Datacenter",
+        "Windows-Server-Technical-Preview"
+      ],
+      "metadata": {
+        "description": "Set this value for the VM Windows OS Version"
+      }
+    },
+    "appPublicIPDnsName": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Set this value for the dns name of the public ip"
+      }
+    },
+    "appDSCUpdateTagVersion": {
+      "type": "string",
+      "defaultValue": "1.0",
+      "metadata": {
+        "description": "This value must be changed from a previous deployment to ensure the extension will run"
+      }
+    },
+
+    "appWebPackage": {
+      "type": "string",
+      "defaultValue": "https://computeteststore.blob.core.windows.net/deploypackage/deployPackage.zip?sv=2015-04-05&ss=bfqt&srt=sco&sp=r&se=2099-10-16T02:03:39Z&st=2016-10-15T18:03:39Z&spr=https&sig=aSH6yNPEGPWXk6PxTPzS6fyEXMD1ZYIkI0j5E9Hu5%2Fk%3D",
```

Comment:
I assume this default value is meant to be here, and that it's safe? i.e. that this link doesn't give people write access to do malicious things to something that exists?